static/js/browser2/out_arc_section.tsx

```diff
@@ -40,7 +40,6 @@ const IGNORED_OUT_ARC_PROPERTIES = new Set([
   "geoJsonCoordinatesDP2",
   "geoJsonCoordinatesDP3",
   "censusACSTableId",
```

Comment:
why are we filtering censusACSTableId?

Reply:
I think that was something I had added by accident before, I will remove that too.